ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -191,10 +191,6 @@ tests:
   commands: TEST_SUITE=openshift/conformance/parallel run-tests
   openshift_installer:
     cluster_profile: aws
-- as: e2e-azure
```

Comment:
Not sure I understand this part... does it mean `/test e2e-azure` stops working? How do you run the azure tests?

Reply:
If I understand correctly, it means that prowgen no longer generates the job in `openshift-origin-master-presubmits.yaml`, so I can modify it without prowgen overriding the modifications.  Without this change, the generated-config job would fail on `openshift-origin-master-presubmits.yaml` with a diff showing that my modifications were being reverted, and the rehearse job would reflect that reversion.